content/index.html

```diff
@@ -63,4 +63,9 @@ <h2>Get involved!</h2>
 		</div>
 	</div>
 	<div class="clear"></div>
-</div>
\ No newline at end of file
+</div>
+<% # winter easter egg from december to february ;)
+   if Date.today.next_month.month <= 3 %>
+<script src="/js/vendor/jquery.flurry.min.js"></script>
+<script src="/js/snowflakes.js"></script>
```

Comment:
;)
Because this is a ruby directive: How do you ensure that this is disabled after February, manual recompile?

Reply:
Yeah, I want to have a systemd timer that regenerates the homepage once a month. Which can then be triggered from jenkins when there are changes.